src/Compilers/CSharp/Portable/Errors/MessageID.cs

```diff
@@ -193,6 +191,10 @@ internal static LanguageVersion RequiredVersion(this MessageID feature)
             // Checks are in the LanguageParser unless otherwise noted.
             switch (feature)
             {
+                // C# 8 features.
+                case MessageID.IDS_FeatureStaticNullChecking:
```

Comment:
Is this error produced from parsing or binding? If from binding, I think we're supposed to put a comment here.

Reply:
Errors are produced in parsing.